.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
   "editor.formatOnSave": false,
-  "mocha.files.glob": "test/unit/**/metadata.js"
+  "mocha.files.glob": "test/unit/**/await_write_finish.js"
```

Comment:
Is it really meant to be published?

Reply:
no, it i an error. Even the previous version (but I did not revert the commit of this file `.vscode/settings.json`.